openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/StreamProcessor.java

```diff
@@ -345,7 +349,7 @@ private void startSharing(Client c, Optional<StreamDesc> osd, JSONObject msg, Ac
 		}
 	}
 
-	private void pauseSharing(Client c, String uid) {
+	void pauseSharing(Client c, String uid) {
```

Comment:
Needed package private access for verify via Mockito if the method has been called.